### PR TITLE
Bumping application version to v1.95.1 and bumping pgvecto-rs

### PR DESCRIPTION
**Pull request by samholton**
Updating application version to latest and adjusting pgvecto-rs. Also added example comments for mounting a read-only external library from NFS

### CODE REVIEW

**Comment by prometheanfire** [comment]
I suppose the migration plan we should do is to manually kill containers when upgrading (kinda expected, just asking)

**Comment by samholton** [comment]
> I suppose the migration plan we should do is to manually kill containers when upgrading (kinda expected, just asking)

Yeah, I upgraded from ` v1.94.1` and just needed to restart the postgres pod once after it upgraded

**Review by mertalev — COMMENTED** [review]
The version bump looks good, but I'll defer to @bo0tzz on the external library example

File: charts/immich/values.yaml [hunk 1]
```diff
@@ -70,12 +70,26 @@ server:
           paths:
             - path: "/"
       tls: []
+  # persistence:
+  #   external-library:
+  #     enabled: true
+  #     readOnly: true
+  #     type: nfs
+  #     server: your-server
+  #     path: /mnt/to/photos
```
[the remark this answers is not in the document]
Comment (djjudas21) [reply charts/immich/values.yaml:79]: Not technically related to the PR, but as it's just the addition of a comment it's probably not worth splitting it out

[the remark this answers is not in the document]
Comment (bo0tzz) [reply charts/immich/values.yaml:79]: This is unrelated to the PR, we don't need to document options that are already properly documented by common-library, and finally the version of the lib that we use does not support `type: nfs` ;)

**Review by bo0tzz — CHANGES_REQUESTED** [review]
Please remove the unrelated changes

File: charts/immich/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (samholton) [reply charts/immich/values.yaml:79]: > the version of the lib that we use does not support `type: nfs`

Interesting, when I was first trying to spin up immich to test, I wanted to use a readonly external library to ensure it didn't change any files on TrueNAS Scale. As someone new to Immich, it was not quick/easy to do this. However, the above did work to access the files, so perhaps the common library is missing documentation or its not easy to find.

It was also not immediately apparent that both containers required access to the files. Not hard to debug, but also not easy and user friendly as a new user.

**Review by bo0tzz — APPROVED** [review]

**Pull request merged**